doc/margin-calibration-howto.md

```diff
@@ -0,0 +1,58 @@
+# How to calibrate the label margins of DYMO printers
+
+The goal of calibration is to determine:
+
+1. How many pixels does the print head have?
```

Comment:
3rd goal: Determine active area size of the printhead (mm)

Reply:
What do you mean exactly? Do you mean that we should measure the height of the pattern in mm so that we can compute `px_per_mm`?

Reply:
Exactly. To make sense, this needs to be done on the largest supported tape/label size.